scripts/ASTImportTest.sh

```diff
@@ -37,10 +37,10 @@ fi
 # $1 name of the file to be exported and imported
 # $2 any files needed to do so that might be in parent directories
 function testImportExportEquivalence {
-    if $SOLC $1 $2 > /dev/null 2>&1
+    if $SOLC "$1" $2 > /dev/null 2>&1
```

Comment:
Why not quote `$2`?

Reply:
`$2` contains a list of files separated with spaces. Quoting it would break the script.

I have a different solution to the warning here - Bash arrays - but that will be in a separate PR since it's not just a simple change in quoting.

Reply:
I have just pushed the PR that contains a solution for this one: #10877. Note that this script won't pass shellcheck until the last PR in this set (#10879).